tests/UI/pages/BO/international/taxes/taxRules/index.js

```diff
@@ -313,6 +315,25 @@ class TaxRules extends BOBasePage {
     await this.clickAndWaitForNavigation(page, this.bulkDeleteLink);
     return this.getTextContent(page, this.alertSuccessBlock);
   }
+
+  /**
+   * Enable / disable tax rules by Bulk Actions
+   * @param page
+   * @param enable
+   * @returns {Promise<string>}
+   */
+  async changeEnabledColumnBulkActions(page, enable = true) {
+    // Select all rows
+    await this.bulkSelectRows(page);
+
+    // Click on Button Bulk actions
+    await page.click(this.bulkActionMenuButton);
+
+    // Click to change status
+    await this.clickAndWaitForNavigation(page, enable ? this.bulkEnableLink : this.bulkDisableLink);
+    /* Successful message is not visible, skipping it */
+    //return this.getTextContent(page, this.alertSuccessBlock);
```

Comment:
Same here, we need to keep track of this.

Reply:
Okey :+1: